### PR TITLE
Jordan/tx flow love

### PR DESCRIPTION
**Pull request by jbibla**
For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -379,6 +379,10 @@ export default {
         body: `You have successfully completed a transaction.`
       })
     },
+    featureFlag: {
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue]: nice!

File: src/ActionModal/utils/ActionManager.js [hunk 2]
```diff
@@ -81,7 +81,7 @@ export default class ActionManager {
     const graphqlHost = urlParams =>
       (urlParams.graphql ? decodeURIComponent(urlParams.graphql) : false) ||
       config.graphqlHost
-    const urlParams = getURLParams(window)
+    const urlParams = `http://localhost:4000`
```
Comment (faboweb) [inline src/ActionModal/utils/ActionManager.js]: why? o.O

Comment (jbibla) [reply src/ActionModal/utils/ActionManager.js]: lol i reverted nothing to worry about

File: src/ActionModal/components/ActionModal.vue (hunk 1 above)
Comment (jbibla) [reply src/ActionModal/components/ActionModal.vue]: not really :/  
doesn't feel right to me.

**Pull request merged**